src/AccessToken/Verify.php

```diff
@@ -60,7 +60,7 @@ class Verify
     /**
      * @var \Firebase\JWT\JWT
      */
-    public $jwt;
+    public JWT $jwt;
```

Comment:
not necessary for the bug fix
```suggestion
    public $jwt;
```